bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -217,6 +219,10 @@ bool PreProcessor::checkClientMsgCorrectness(const ClientPreProcessReqMsgUniqueP
 }
 
 void PreProcessor::updateAggregatorAndDumpMetrics() {
+  if (myReplica_.getCurrentView() != lastViewNum) {
```

Comment:
Let's clean the map, in any case, periodically. Let's say, every hour.

Reply:
done